lang/fr.json

```diff
@@ -15,12 +15,26 @@
   "Playback Rate": "Vitesse de lecture",
   "Subtitles": "Sous-titres",
   "subtitles off": "Sous-titres désactivés",
-  "Captions": "Sous-titres",
-  "captions off": "Sous-titres désactivés",
+  "Captions": "Sous-titres transcrits",
+  "captions off": "Sous-titres transcrits désactivés",
   "Chapters": "Chapitres",
+  "Close Modal Dialog": "Fermer la boîte de dialogue modale",
+  "Descriptions": "Descriptions",
+  "descriptions off": "descriptions désactivées",
+  "Audio": "Audio",
```

Comment:
We've actually decided that we only need `Audio Track` right now, so, this line is not necessary.